package.json

```diff
@@ -66,7 +67,8 @@
     "husky:pre-push": "yarn lint:commit",
     "build-storybook": "lerna run build-storybook --stream --scope docs-examples",
     "prepare": "husky install",
-    "ts:check": "lerna run ts:check --stream"
+    "ts:check": "lerna run ts:check --stream",
+    "postinstall": "yarn build:node-scripts"
```

Comment:
This post install hook had to be added because the TS based node scripts `ui-build` have to be transpiled before we are able to run any other package json script, because they heavily depend on it and our current node setup can't run TS on by itself.